### DISCUSSION TITLE
[Projects Beta] Multiple select actions

### DISCUSSION THREAD

**Discussion by andre-pdassis** [community/community#8960] [Projects and Issues] [answered]
It would be interesting to shift+click multiple tasks and be able to execute a common action to all of them, like update the status or change assignee.

Update item by item in a large project feels cumbersome.

Edit: As pointed by @angalic there's a workaround for multiple selections. The full answer can be viewed at #6528 

> be sure to shift/cmd/cntrl click on the rows via the row numbers on the far left side

**Comment by MarkBennett** [comment]
Agreed. A few times we've had to reprioritize work, and it would be nice to be able to grab groups of issues at once and drag them around.

**Comment by techwithmatheus** [comment]
Also, for example, I just created a bunch of new tasks up front and now I'm having to set their initial status individually to "to do" which is a major pain.

**Comment by iba-1** [comment]
This looks like a very basic feature for such a management project, is there any way it can one day see the light?

**Reply by andre-pdassis** [reply]
I hope so.

**Comment by jugglingcats** [comment]
This should really be split into multiple enhancements. We're mostly interested in multiple select and then drag to re-order the board, as Mark Bennett mentioned.

**Comment by RengokuDK** [comment]
Agree. We have the process to Archive issues when the Milestone completes. However this means right now that we need to select one-by-one and click Archive. Would really like to be able to Multi-select and then chose Archive.
Maybe if Project workflow supported 'When a Milestone is closed then archive all CLOSED issues

**Reply by angalic** [reply]
multiselect and archive or delete is already possible, see: https://github.com/community/community/discussions/6528

**Reply by MarkBennett** [reply]
Interesting. Nice that it's possible, but it would be better if there was some affordance in the UX to make it more discoverable for users.

Thanks for pointing it out regardless @angalic! :)

**Comment by bryanwhiting** [comment]
Any update on this? first thing I googled when starting with projects.

**Comment by bryanwhiting** [comment]
At least you can multi-assign in the github issues.

**Comment by rileybroughten** [comment]
👋 Hello! There may be some overlap in this discussion with https://github.com/orgs/community/discussions/5465, so feel free to add similar thoughts there as well! Some additional notes:
- bulk-editing in tables is currently supported for copying/pasting cell values for a single or range of cells, check out the announcement [here](https://github.blog/changelog/2023-04-06-github-issues-projects-april-6th-update/#t-rex-bulk-editing-in-tables) and know there is more to come!
- bulk-edit in boards is coming soon 👀 check out the [public roadmap item](https://github.com/github/roadmap/issues/762) for selecting multiple items and moving them between columns

Regarding other ways to bulk-edit items, what else would you like to see? What fields are you most often wanting to change in bulk?

**Reply by rileybroughten** [reply, marked as answer]
Bulk drag and drop for board items is now supported - changelog announcement [here](https://github.blog/changelog/2023-05-25-github-issues-projects-may-25th-update/#%F0%9F%92%AA-%E2%8C%A8%EF%B8%8F-bulk-updates-and-keyboard-navigation-on-boards) 😁

**Reply by andre-pdassis** [reply]
Amazing!

**Reply by dondelicaat** [reply]
@rileybroughten Would be great to see the same support for tables. Frequently use this (and also cmd-clicking multiple distinct stories) to drag and drop stories between sprints in bulk in Jira. It would make the decision to switch to github projects easier with this in place.

**Reply by everhardt** [reply]
What this doesn't yet allow is moving a group of tickets to another position in the same column, eg. to the top.

**Reply by josephine-njooi** [reply]
Looks like a separate discussion https://github.com/orgs/community/discussions/166637 has been created for the fact that we can't yet move multiple tickets to another position within the same column. This is really frustrating to have to reorder tickets one by one.
